### DISCUSSION TITLE
Can I use markerHandler in MobileCalendar too?

### DISCUSSION THREAD

**Discussion by MarvinMuuss** [taiga-family/taiga-ui#3651] [Q&A] [answered]
Hi there,
I've recently discovered this amazing library, thanks for your efforts!
I'm currently on my way to implement the first version of a new application and I'm currently stuck with using the mobileCalendar component which I want to feed the markersHandler into.
According to the API specs this is not possible, which would be an absolute bummer.

Is there maybe a way to enable this for the mobile calendar too?

If you're interested this is the function I use to open the dialog for the datepicker
```
openCalendarDatePicker(): void {
    let control = new FormControl(new TuiDay(this.selected.year(), this.selected.month(), this.selected.date()));
    const dataStream = tuiControlValue(control);
    const computedInjector = Injector.create({
      providers: [
        {
          provide: TUI_CALENDAR_DATE_STREAM,
          useValue: dataStream,
        },
      ],
      parent: this.injector,
    });
    const content = new PolymorpheusComponent(
      TuiMobileCalendarDialogComponent,
      computedInjector,
    );
    const ONE_DOT: [string] = ['var(--tui-success-fill)'];
    let dialog$: Observable<TuiDay> = this.dialogService.open(content, {
      size: `fullscreen`,
      closeable: false,
      data: {
        min: new TuiDay(2020, 1, 1),
        markerHandler: (day: TuiDay) => {
          // Attention: do not create new arrays in handler, use constants intead
          return day.day % 2 === 0 ? ONE_DOT : ONE_DOT
        }
      },
    });
    dialog$.subscribe((value:TuiDay) => {
      this.store.dispatch(new SelectDate({date: dayjs(value.toUtcNativeDate())}));
      this.cdk.detectChanges();
      this.customSwiper.swiperRef.slideNext(1)
      this.customSwiper.swiperRef.slidePrev(1)
    });
  }
```

**Comment by waterplea** [comment, marked as answer]
Hi Marvin. Thank you for kind words. You can open an issue to add this feature, I believe it shouldn't be that hard. Although chances are, mobile calendar would be heavily reworked in the future as we plan on adding more mobile specific behavior to components, as well as overhauling current attempts.
